docs/technical/new_instance.md

```diff
@@ -0,0 +1,188 @@
+# Creating a new Insights instance
+The overall idea is strong isolation across districts, which means some initial setup overhead.  Here's how you do it!  More detailed notes for various steps are below.
+
+### 1: Stand up an instance
+Goal: Visit the home page signed-out
+- Add domain
+- SSL certificate
+- New Heroku app with Node/Ruby buildpacks, grant access to team
+- Review and set environment variables (minimally)
+- Review and set PerDistrict values (minimally)
+- Review and add Heroku scheduler jobs (minimally)
+- Set up Postgres and seed database (minimally)
+- Set up Logentries
+- Set up Rollbar
+- Set up Mixpanel
+- Set up CSP and CSP logger
+
+### 2: Sign in
+Goal: Sign in as developer, see empty home page, roster page, districtwide pages
+- Create developer user
+
+### 3: Import first data
+Goal: Sign in as districtwide educator, use the home page, roster page, student profile page, take notes for students in support meetings
+- Create an instance for receiving data (maybe with static IPs, grant access to team)
+- `EducatorsImporter` and validate
+- `StudentsImporter` and validate
+- Review permissions with district partner
+- Set up Mailgun and enable usage emails 
+
+### 4: Enable additional features, or ensure disabled
+Goal: Educators using additional feature (eg, "insights box" for absences) or using additional data within support meetings (eg BBST).
+- Student photos (with AWS config, grant access to team)
+- IEP PDFs (with AWS config, grant access to team)
+- `AttendanceImporter` and validate
+- `BehaviorImporter` and validate
+- ACCESS and validate
+- Next Gen MCAS and validate
+- Old school MCAS and validate
+- DIBELS and validate
+- STAR and validate
+- HS Courses and Sections for authorization
+- HS grades
+- Class lists
+- Review `PerDistrict`
+
+### 5: Final review
+- Review ENV variables
+- Review `PerDistrict`
+- Review scheduled jobs
+- Review Puma tuning (eg, WEB_CONCURRENCY)
+- Load test or scale accordingly
+
+
+# Details
```

Comment:
When I read through I don't see much logical flow between the checklists above and the "Details" section below. What do you think about mapping out from each checklist (or each step, if it's a major step) to a relevant "Details" doc or section?

Reply:
Yeah there's no connection, I'll write that explicitly :)  I think we need more "a list" than we do a user-friendly tutorial.  How about next time you and I sit down to actually do this work, we can just scan through the bottom of this doc and find what we need, and update this based on our experience from the next district?